### ISSUE TITLE
Double closing tag

### ISSUE THREAD

**Issue by rcorsari** [BalduinLandolt/vs-code-surround-with-tag#5]
Hello I suffer a conflict with maybe some built-in function of Visual Studio Code

I have not so many extensions and I have disabled them all, but, if I'm in some HTML code and I e.g. select a word and with Ctrl-i I add e.g. the STRONG tag I get this result

`<strong></strong>mySelectedWord</strong>`

and the cursor is found positioned between the `<strong>|</strong>` first two tags

It is obvious that something does recognize the opening tag and automatically adds the closing tag right after

Can you hint what can be the source of this annoying issue?

Thank you

```
Version: 1.62.3 (user setup)
Commit: ccbaa2d27e38e5afa3e5c21c1c7bef4657064247
Date: 2021-11-17T08:11:14.551Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043
```

**Comment by BalduinLandolt** [comment]
Thank you for reporting the issue. I will have to investigate it, hopefully I can come up with a quick fix.

**Comment by BalduinLandolt** [comment]
I tested it on a clean install of VS Code and the issue still occurred, so it's clearly not a different extension causing the issue.  
Also, this only happens in HTML files, not in XML. So my guess is it's VS Code's built-in HTML support recognizing the tag and auto-closing it.

A (very cheap) workaround I found: If you do the thing, after it auto-added the additional closing tag, you can hit `Ctrl/Cmd+Z` and it will undo the added closing tag. The issue with the wrongly positioned cursor remains, though.

I'll have to see what I can do about this.

**Comment by rcorsari** [comment]
Thank you for the TIP, I use it on some spellchecks in LibreOffice , didn't thought to apply it here, good!

About Visual Studio Code, if you have some channel to issue the "bug" to Visual Studio Code developers... I don't have any

Thank you for the suggestion

**Comment by BalduinLandolt** [comment]
@rcorsari I think I managed to fix it. I will release a new version soon.

If that's ok with you, I'll tag you, asking to verify that the fix worked for you too, as soon as I have released.

**Comment by estufa8** [comment]
Hi.
Looks like it works ok by disabling `HTML: Auto closing tags` in settings.
And then, typing `div`+TAB key still writes `<div></div>` in my case.
Thanks @BalduinLandolt for this useful extension.
Regards.

**Comment by BalduinLandolt** [comment]
@rcorsari  @estufa8 I just released version 1.0 to the market place. With that, the bug should be fixed. (Plus I added some more features...)  
Feel free to give feedback on those, or request more features, if you can think of something useful.

I'm glad you like the extension!
